app/customer.py

```diff
@@ -0,0 +1,71 @@
+import math
+
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: int,
+            fuel_consumption: float
+    ) -> None:
+
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.fuel_consumption = fuel_consumption
+
+    def print_customer_name_and_money(self) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+    def calculate_buyer_balance_of_money(self, money: int) -> None:
+        print(f"{self.name} now has {self.money - money} dollars\n")
+
+    def get_price_product_cart(self, shop: Shop, file_config: dict) -> int:
+        distance = (
+            math.sqrt(math.pow((self.location[0] - shop.location[0]), 2)
+                      + math.pow((self.location[1] - shop.location[1]), 2))
+        )
+
+        price_of_all_products = sum(
+            [value * shop.products[key]
+                for key, value in self.product_cart.items()
+                if key in shop.products]
+        )
+
+        price_fuel = (
+            (distance * 2 / 100 * self.fuel_consumption)
+            * file_config["FUEL_PRICE"]
+        )

```

Comment:
The calculation of `price_fuel` should account for the round trip (to the shop and back home). Currently, it only calculates the cost for a one-way trip. Multiply the distance by 2 to include the return trip as well.